WearStandaloneGoogleSignIn/README.md

```diff
@@ -16,8 +16,7 @@ adjust them according to your environment.
 Support
 -------
 
-You can find support by posting up on the Android Wear Developers Community
-page at the link below:
+- Stack Overflow: http://stackoverflow.com/questions/tagged/android
```

Comment:
Maybe use the wear-os tag?

Reply:
can follow up in a separate PR and change for all projects 👍 

`ComposeAdvanced` actually [lists](https://github.com/android/wear-os-samples/blob/468b0ffa6dbda6f035162a0c29b891d77b791e5d/ComposeAdvanced/README.md?plain=1#L66-L67) both tags, any preference?

Reply:
Wear-os is the preference for me

Reply:
https://github.com/android/wear-os-samples/pull/601